packages/web/src/components/premium-content-purchase-modal/components/PurchaseSummaryTable.tsx

```diff
@@ -6,31 +6,77 @@ import { Text } from 'components/typography'
 const messages = {
   summary: 'Transaction Summary',
   premiumTrack: 'Premium Track',
+  downloadableFiles: 'Downloadable Files',
+  allStemFiles: 'All Stem Files',
```

Comment:
when do we use "All Stem Files" vs "Downloadable Files"? I see "All Stem Files" once in the mocks but I feel like this may be a mistake in design?

Reply:
When stems are available, but the og download is not. Is there no case where that happens?

Reply:
no that is a valid case, just not clear to me from the mocks i guess.

Reply:
resolved offline, dropped this All Stem Files bit